frontend/tests/controllers/IdentityCreateTest.php

```diff
@@ -0,0 +1,233 @@
+
+        $is_creator_member = Authorization::isGroupIdentityCreator($creator_identity->identity_id);
+        // Asserting that user belongs to the  identity creator group
+        $this->assertEquals(1, $is_creator_member);
+
+        $is_creator_member = Authorization::isGroupIdentityCreator($mentor_identity->identity_id);
+        // Asserting that user doesn't belong to the identity creator group
+        $this->assertNotEquals(1, $is_creator_member);
+    }
+
+    /**
+     * Basic test for creating a single identity
+     */
+    public function testCreateSingleIdentity() {
+        // Identity creator group member will upload csv file
+        $creator = UserFactory::createGroupIdentityCreator();
+        $creatorLogin = self::login($creator);
+        $group = GroupsFactory::createGroup($creator, null, null, null, $creatorLogin);
+
+        $identityName = substr(Utils::CreateRandomString(), -10);
+        // Call api using identity creator group member
+        IdentityController::apiCreate(new Request([
+            'auth_token' => $creatorLogin->auth_token,
+            'username' => "{$group['group']->alias}:{$identityName}",
+            'name' => $identityName,
+            'password' => Utils::CreateRandomString(),
+            'country_id' => 'MX',
+            'state_id' => 'QUE',
+            'gender' => 'male',
+            'school_name' => Utils::CreateRandomString(),
+            'group_alias' => $group['group']->alias,
+        ]));
+
+        $response = GroupController::apiMembers(new Request([
+            'auth_token' => $creatorLogin->auth_token,
+            'group_alias' => $group['group']->alias,
+        ]));
+
+        $this->assertEquals(1, count($response['identities']));
+    }
+
+    /**
+     * Test for creating an identity with wrong group
+     * @throws InvalidDatabaseOperationException
+     */
+    public function testCreateIdentityWithWrongGroup() {
+        // Identity creator group member will upload csv file
+        $creator = UserFactory::createGroupIdentityCreator();
+        $creatorLogin = self::login($creator);
+        $group = GroupsFactory::createGroup($creator, null, null, null, $creatorLogin);
+        $wrongGroupAlias = 'wrongGroupAlias';
+        $identityName = substr(Utils::CreateRandomString(), -10);
+        // Call api using identity creator group member
+        try {
+            $response = IdentityController::apiCreate(new Request([
+                'auth_token' => $creatorLogin->auth_token,
+                'username' => "{$wrongGroupAlias}:{$identityName}",
+                'name' => $identityName,
+                'password' => Utils::CreateRandomString(),
+                'country_id' => 'MX',
+                'state_id' => 'QUE',
+                'gender' => 'male',
+                'school_name' => Utils::CreateRandomString(),
+                'group_alias' => $group['group']->alias,
+            ]));
```

Comment:
puedes agregarle el `$this->fail(...)` y quitar el `@throws` de L62?

Reply:
listo